cypress/tests/bucket-policy.spec.ts

```diff
@@ -91,4 +92,50 @@ describe('Bucket policy page', () => {
     cy.byTestID('delete-action').click();
     cy.byTestID(SINGLE_BUCKET_POLICY).should('not.exist');
   });
+
+  // This creates bucket policy from data sources dropdown with cache
+  it('creates Bucket policy with single data source and enabled cache', () => {
```

Comment:
Please have a look into this [PR](https://github.com/red-hat-storage/mcg-ms-console/pull/67/files#diff-fd832311a63c8c6a224aaf766f612c2200ee4d4929da205f3e50dbbff78d43c2) Idea is to bring out common stuff into a separate file as we have to the data source.

@alfonsomthd  what is your opinion regarding this, should we have a common separate file?

Reply:
I agree on code reuse: let's move the common logic to a helper method on the appropriate helper file.